packages/esm-form-entry-app/src/app/openmrs-api/concept-resource.service.ts

```diff
@@ -29,6 +29,15 @@ export class ConceptResourceService {
       );
   }
 
+  public getAllConcepts(): Observable<any[]> {
```

Comment:
Can we give this a better type than `any`? Reason is quite simple - here we say "hey, that's any", but below, e.g., we say "it has an `uuid` field". So I wonder - what else do we know / assume? We should put in code here.

(same for the existing code, e.g., `getConceptByUuid` below - everything we know we should already have there)

Reply:
From the discussion I had with @rbuisson today, I will be removing this, I don't need to fetch concepts because the function will be expecting selected concept uuids, so I am going to remove this.